dev/prod_builders.json

```diff
@@ -82,7 +82,8 @@
       "name": "Linux complex_layout_scroll_perf__devtools_memory",
       "repo": "flutter",
       "task_name": "linux_complex_layout_scroll_perf__devtools_memory",
-      "flaky": false
+      "flaky": true,
+      "issue_url": "https://github.com/flutter/flutter/issues/82741"
```

Comment:
I haven't announced it yet, but I'm aiming to deprecate this JSON config Wednesday night in favor of the `.ci.yaml`. Just commenting here as I'll need to move these to comments in the migration PR (@CaseyHillers @christopherfujino )

Reply:
I think you just announced it. :-)

Reply:
I suppose new properties could be naturally supported in the `.ci.yaml`?

Reply:
I suppose new properties could be naturally supported in `.ci.yaml`?

Reply:
Yes, PRs welcome if you want to add automation around this

Reply:
> I suppose new properties could be naturally supported in `.ci.yaml`?

You would have to update the protobuf in flutter/cocoon first. I actually see this as a feature, as if you add a new target and mispell a field, CI will fail on an unknown field, rather than silently ignoring it.